### DISCUSSION TITLE
Run pipeline in dev/test mode?[general]

### DISCUSSION THREAD

**Discussion by eveyp** [ropensci/targets#1027] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Is there a way to run (`tar_make()`?) a pipeline in a dev/test mode where certain targets don't run?

It seems like `tar_force()` or `tar_skip()` is the way to do this, but I'm having trouble figuring out how I'd pass the operation (production or dev) mode to them.

**Comment by wlandau** [comment]
Cues can control which targets do and don't run, and you can set the `mode` argument programmatically: https://docs.ropensci.org/targets/reference/tar_cue.html.
